layers.go

```diff
@@ -387,6 +387,10 @@ func (r *layerStore) loadMounts() error {
 	}
 	layerMounts := []layerMountPoint{}
 	if err = json.Unmarshal(data, &layerMounts); len(data) == 0 || err == nil {
+		for _, layer := range r.layers {
```

Comment:
Can we add a comment here explaining why it's safe/correct to reset the counter?

Reply:
… why it is necessary. Yes, please.

Reply:
Added.